cmd/swaptester/main.go

```diff
@@ -189,17 +196,23 @@ func getRandomExchangeRate() types.ExchangeRate {
 }
 
 func generateBlocks() {
-	cXMRMaker := monero.NewWalletClient(common.DefaultXMRTakerMoneroEndpoint)
-
+	cXMRMaker := monero.NewThinWalletClient(defaultXMRMakerMoneroEndpoint)
```

Comment:
I changed line above to the XMR Maker, because I didn't understand why the XMR taker would be mining monero.